mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/71 - camera update

Closes #71 

To test:
- Run any scenario
- Drag with left click to pan camera
- Drag with right click to rotate camera
- Use scroll wheel to zoom in and out
- Make sure camera controls work and feel correct (direction and speed of camera movement)
